### DISCUSSION TITLE
Tycho 2.4.0 mentions new mojo but link is dead

### DISCUSSION THREAD

**Discussion by tivervac** [eclipse-tycho/tycho#370] [Q&A] [answered]
> A new mojo tycho-gpg-plugin:sign-p2-artifacts-mojo was added to add GPG signatures to artifacts metadata. Those GPG signatures are later expected to be used by p2 during installation to verify integrity and build trust in installed components.

The following link is dead https://www.eclipse.org/tycho/sitedocs/tycho-gpg-plugin/sign-p2-artifacts-mojo.html

Any place I can find a living version?

**Comment by mickaelistria** [comment, marked as answer]
Unfortunately, this mojo wasn't properly added to build for 2.4.0. You can try 2.5.0-SNAPSHOT instead https://github.com/eclipse/tycho/blob/master/CONTRIBUTING.md#try-snapshots-and-report-issues

**Reply by tivervac** [reply]
Hmm, why 2.5.0-SNAPSHOT? 2.5.0 has already been released hasn't it? Did you mean 2.6.0-SNAPSHOT?

**Reply by mickaelistria** [reply]
Yes, 2.6.0-SNAPSHOT, sorry.
